package.json

```diff
@@ -10,10 +10,8 @@
     "cloud-env": "0.2.2",
     "http-proxy": "1.16.2",
     "nodemailer": "4.0.1",
-    "node-static": "0.7.9"
-  },
-  "nonDefaultDependencies": {
```

Comment:
I don't know if it was ever a real thing? We just intentionally did that because we didn't want npm to install it automatically, which it does for optional dependencies.